hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/table/HoodieFlinkCopyOnWriteTable.java

```diff
@@ -265,6 +266,20 @@ public void rollbackBootstrap(HoodieEngineContext context, String instantTime) {
     throw new HoodieNotSupportedException("Bootstrap is not supported yet");
   }
 
+  /**
+   * TODO :
+   * Refactor {@link FlinkCleanActionExecutor} to support scheduling of cleaning.
+   * @param context HoodieEngineContext
+   * @param instantTime Instant Time for scheduling cleaning
+   * @param extraMetadata additional metadata to write into plan
+   * @return
+   */
+  @Override
+  public Option<HoodieCleanerPlan> scheduleCleaning(HoodieEngineContext context, String instantTime, Option<Map<String, String>> extraMetadata) {
```

Comment:
this does not seem ok to do?

Reply:
Yeah, had a TODO on this, addressed it now.